modules/local/qiime2_ancom_tax.nf

```diff
@@ -43,7 +43,7 @@ process QIIME2_ANCOM_TAX {
         --to-tsv
 
     if [ \$(grep -v '^#' -c ${table.baseName}-level-${taxlevel}.feature-table.tsv) -lt 2 ]; then
-        echo ${taxlevel} > ancom/\"WARNING Summing your data at taxonomic level ${taxlevel} produced less than two rows (taxa), ANCOM can't proceed -- did you specify a bad reference taxonomy?\".txt
+        echo ${taxlevel} > ancom/"${table.baseName}_WARNING_Summing_your_data_at_taxonomic_level_${taxlevel}_produced_less_than_two_rows_(taxa)_ANCOM_cant_proceed_--_did_you_specify_a_bad_reference_taxonomy.txt"
```

Comment:
Prepending `${table.baseName}` unfortunately defeats its purpose, because the warning wont be picked up by https://github.com/nf-core/ampliseq/blob/e66c9d371dcbbd3cc6b91a1dec93daffaef52c16/subworkflows/local/qiime2_ancom.nf#L45 any more. Would need to be adjusted.

Reply:
Right, I'll put it after the `WARNING`.

Reply:
`it.baseName[0].toString().replace("WARNING ","QIIME2_ANCOM_TAX: ")` will still not work, because its now `WARNING_` instead of `WARNING ` (mind the space), correct? Other than that the PR seem fine to me.

Reply:
OK, I reverted most of this, just keeping the `table.baseName` in the filename.